modules/azure/loganalytics.go

```diff
@@ -0,0 +1,68 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/operationalinsights/mgmt/2020-03-01-preview/operationalinsights"
+)
+
+// LogAnalyticsWorkspaceExists indicates whether the operatonal insights workspaces exists; otherwise false or error
+func LogAnalyticsWorkspaceExists(workspaceName, resourceGroupName, subscriptionID string) bool {
+	ws, err := GetLogAnalyticsWorkspaceE(workspaceName, resourceGroupName, subscriptionID)
+	if err != nil {
+		return false
+	}
+
+	return (*ws.Name == workspaceName)
```

Comment:
need to handle when workspace name is invalid.  Please follow pattern in AvailabilitySet https://github.com/gruntwork-io/terratest/blob/48e24df893c925adef331c342b7d3979da08f51c/modules/azure/availabilityset.go#L21 @davesee